doc/install.rst

```diff
@@ -27,21 +27,37 @@ Installation
 Python 3 (>=3.6) must be installed. Other dependencies will be pulled in with
 the instructions below.
 
-IOData can be installed with conda:
+To install IOData using conda package management system, install
+`miniconda <https://conda.io/miniconda.html>`__ or
+`anaconda <https://www.anaconda.com/download>`__ first, and then:
 
 .. code-block:: bash
 
-    conda install theochem::iodata
+    # Activate your main conda environment if needed.
 
-It can also be installed with pip. One of the following is fine, whichever you
-prefer:
+    # Create a horton3 conda environment. (optional, recommended)
+    conda create -n horton3
+    source activate horton3
+
+    # Install the stable release.
+    conda install -c theochem iodata
+
+    # For developers:
+    # Install the testing release. (beta)
+    conda install -c theochem/label/test iodata
+    # Install the development release. (alpha)
+    conda install -c theochem/label/dev iodata
+
+To install IOData with pip, you may want to create a `virtual environment`_,
+and then:
 
 .. code-block:: bash
 
-    pip install iodata
-    pip install iodata --user
-    python3 -m pip install iodata
-    python3 -m pip install iodata --user
+    # Install the stable release.
```

Comment:
I would suggest leaving the option with --user as an extra option not depending explicitly on having a conda environment and installing as a user in the home directory. Not all users might be aware of this option.

Reply:
Agreed. Also the `python3 -m pip install` should be mentioned somehow. That one can be a workaround when the PATH variable is incomplete, which would be nice to explain.

Reply:
This is fixed.